docs/Rapid/4-Keyper Manual/4-Adaptive Designer/6-How to configure connection with data table/6-How to configure connection with data table.md

```diff
@@ -0,0 +1,132 @@
+
+The connection will be visible in the Overview tab's Connections section.  The title of the connection is the title of the Data Table.
+
+![Image showing normal table connection in canvas](<Root Connection 5.png>)
+
+### How to create a child connection?
+
+To create a child connection under a parent connection, please consider the following steps:
+
+1. In the Overview tab's Connections section, click on the title of the connection. This will open a section in the middle of the canvas. This section displays the properties of the parent select.
+
+Observe there is a Children section, which has three options.
+
+![Image showing Parent properties section](<Child connection 1.png>)
+
+  :::info[Please note]   
+  There are three types of children connections that can be created here. 
+
+    1. **UNIVERSAL LINK** - This is a two way link between the connecting tables' items. For example - let the parent be Tasks table and child be Employees table. If the connection is established on Task Assignee column, then the system will fetch / feed employee assigned to the specific task AND also can fetch / feed all the tasks where the assignee is a specific employee. It thus becomes a two-way link and a many-to-many relationship (based on data).
+
+    2. **MULTILOOKUP** - This is a one way link between the connecting tables' items and the system can fetch multiple items of the child connected to the parent. For example - let the parent be Employees table and child be the Tasks table. If the connection is established on Task Assignee column, then the system can fetch / feed all the Tasks where the assignee is a particular employee **but** cannot access employee details from tasks table. Therefore, it is a one way, one-to-many relationship.
+
+    3. **LOOKUP** - This is a also one way link between the connecting tables' items and the system can fetch a single item of the child connected to the parent. For example - let the parent be Tasks table and child be the Employees table. If the connection is established on Task Assignee column, then the system can fetch / feed only one employee who is assigned to the particular task **but** cannot access other task details from Employees table. Therefore, it is a one way, one-to-one relationship.
+:::
+
+
+2. Click on one of the three options based on the type of child connection you wish to establish. 
+
+This will open a section in the middle of the canvas. 
+
+![Image showing different fields to fill for creating child connection](<Child connection 2.png>)
+
+3. Fill-out the details which will define the behaviour of the child connection. consider the following:
+
+   a. **Source** - The source will be pre-selected here based on the parent.  
+   b. **Nature of connection** - Select View (if you want an array of items) or Single Item.  
```

Comment:
This is called the Method. The connection method is view. 

I like "Nature of connection" as an explanation, though. 
```suggestion
   b. **Method** - The method defines the nature of connection. Select View (if you want an array of items) or Single Item.  
```